company/how-to-guides-for-staff/how-to-purchase/how-to-on-board-as-a-vendor/how-to-get-paid.md

```diff
@@ -2,61 +2,62 @@
 
-_Example 2_: a staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his LLC titled Better Coding, John Smith must have the following banking institution setup:
+_Example 2_: A staff member named John Smith is located in Spain and signed a consulting agreement with Mattermost, Inc. under his LLC titled Better Coding. John Smith must have the following banking institution setup:
 
 * The bank account is located in Spain
 * The bank account is in the name of Better Coding, LLC
 
-If you provide information for your banking institution, your invoice and/or expense reimbursements will be made through Bill.com. As noted above, Mattermost requires banking information from a primary banking institution to be able to make reimbursements through Bill.com. Approved expense reports are reimbursed either by the 15th or end of each month. 
+If you provide information for your banking institution, your invoice and/or expense reimbursements will be made through TransferWise. As noted above, Mattermost requires banking information from a primary banking institution to be able to make reimbursements through TransferWise. Approved expense reports are reimbursed either by the 15th or end of each month.
 
-TransferWise
+##### TransferWise
 
-If you provide information for your TransferWise account, your invoice and/or expense reimbursements will be made through TransferWise. As noted above, you will need to be a staff member located outside of the United States, Canada, United Kingdom and Germany to be able to choose this option. If your consulting agreement is other than your name such as an LLC or LLP, than we request you set up a business account for contracting entity in TransferWise under the same legal name. Mattermost primarily makes payments in USD, unless otherwise stated on your agreement signed with Mattermost. You will receive your expense reimbursement no later than 15th and end of each month. Approved expense reports are reimbursed either by the 15th or end of each month. 
+If you provide information for your TransferWise account, your invoice and/or expense reimbursements will be made through TransferWise. As noted above, you will need to be a staff member located outside of the United States, Canada, United Kingdom, and Germany to be able to choose this option. If your consulting agreement is other than your name such as an LLC or LLP, than we request you set up a business account for contracting entity in TransferWise under the same legal name.
 
-Refer to the [Bank Form](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-get-paid#banking-form) section for guidance on registering so you get paid.
+Mattermost primarily makes payments in USD, unless otherwise stated on your agreement signed with Mattermost. You'll receive your expense reimbursement no later than 15th and end of each month. Approved expense reports are reimbursed either by the 15th or end of each month.
```

Comment:
```suggestion
Mattermost primarily makes payments in USD, unless otherwise stated on your agreement signed with Mattermost. You'll receive your expense reimbursement no later than the 15th and the end of each month. Approved expense reports are reimbursed either by the 15th or the end of each month.
```